### ISSUE TITLE
Als persoon, wiens gegevens aanwezig zijn in de adresseringsfunctie wil ik expliciet toestemming kunnen geven of deze kunnen intrekken voor het bevraagbaar maken van mijn gegevens

### ISSUE THREAD

**Issue by NellyAnbeek** [minvws/generiekefuncties-adressering#68]
### Gebruiker

Persoon

### Wat wil de gebruiker doen?

Als persoon, wiens gegevens aanwezig zijn in de adresseringsfunctie wil ik expliciet toestemming kunnen geven of deze kunnen intrekken voor het bevraagbaar maken van mijn gegevens

### Motivatie

x

### Extra informatie

x

**Comment by stevenvegt** [comment]
Relevantie met:
* #93 
* #92

**Comment by VWSLANS** [comment]
Is een organisatievraagstuk niet voor de adresseringsfunctie

**Comment by BertineUithoven** [comment]
Klopt, dat als er aanleverende bronregisters zijn dit daar moet plaatsvinden.

**Comment by sergejvm** [comment]
Dit kunnen we faciliteren, maar we weten nog niet in hoeverre dit ook in scope komt en zo ja hoever. Is geen archtiectuurvraagstuk, maar we hebben nog niet helemaal uitgewerkt of zorgverleners er (nu) een plek in krijgen. We moeten de discussies die daarover lopen nog voeren.
